README.md

```diff
@@ -705,34 +720,6 @@ List information of all current raids
 
 ### tracking
 
-`/raid tracking [<start|stop|restart|realtime>]`
-
-Start a raid tracking to gather statistics from ongoing raids
-
-
-## info
-
-### world ticking_order
-
-`/info world ticking_order`
-
-Show the ticking order of current dimensions in the game
-
-
-## lifetime
-
-A tracker to track lifetime and spawn / removal reasons from all newly spawned and dead entities
-
-This tracker is mostly used to debug mobfarms. It aims to track the process from mob starting affecting the mobcap to mob being removed from the mobcap. The spawning tracking part of it doesn't cover every kind of mob spawning reasons
-
-Other than being removed from the world, if a mob becomes persistent for the first time like nametagged or item pickup, it will be marked as removal too. If a mob doesn't count towards the mobcap when it spawns, it will not be tracked
-
-This tracker also tracks lifetime of items and xp orbs from mob and block drops as an additional functionality. Note that it doesn't track all item / xp orb spawning, so you'd better have a test before actually using it
-
-Adding a `realtime` suffix to the command will turn the rate result from in-game time based to realtime based
-
-### tracking
-
```

Comment:
looks like you have messed up the readme structure. plz check the orignial readme and fix it